### PR TITLE
add mathtext capabilities to div widget

### PR DESCRIPTION
**Pull request by IuryPiva**
- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)
image

### CODE REVIEW

File: bokeh/models/widgets/markups.py [hunk 1]
```diff
@@ -103,6 +107,12 @@ class Div(Markup):
     The default value is ``False``, meaning contents are rendered as HTML.
     """)
 
+    render_as_mathtext = Bool(False, help="""
```
Comment (bryevdv) [inline bokeh/models/widgets/markups.py:110]: suggest framing this as `enable_mathtext` which "only applies when rendering HTML (that is, when `render_as_text` is False)"

File: bokeh/embed/bundle.py [hunk 2]
```diff
@@ -414,7 +414,16 @@ def _use_mathjax(objs: Sequence[Model | Document]) -> bool:
         bool
     '''
     from ..models.text import MathText
-    return _any(objs, lambda obj: isinstance(obj, MathText))
+
+    def model_require_mathjax(model: Model) -> bool:
+        ''' If any model explicit require MathJax with render_as_mathtext option set to true
+        '''
+
+        for key, value in model._property_values.items():
```
Comment (philippjfr) [inline bokeh/embed/bundle.py:422]: Iterating over every property on every model seems potentially expensive, so while I wouldnt usually recommend getattr I think I'd use some variation of `getattr(model, "render_as_mathtext", False)` here.

Comment (IuryPiva) [reply bokeh/embed/bundle.py]: Just updated it, is this similar to what you are saying? 

https://github.com/bokeh/bokeh/blob/1d8460aab11b858326be8c7a379e76d936a85d5d/bokeh/embed/bundle.py#L418-L424

Comment (philippjfr) [reply bokeh/embed/bundle.py]: That seems totally fine, I probably would have just done: `return  properties.get('enable_mathtext', False)`

Comment (IuryPiva) [reply bokeh/embed/bundle.py]: That looks clean! Thanks!

**Comment by IuryPiva** [comment]
For finishing this one. I need to make MathJax fonts available with our bokeh-mathjax bundle, can someone point me in the right direction here? The same is pending on #11558

**Comment by IuryPiva** [comment]
superseded by #11585

**Pull request closed without merging**